scripts/monitoring/cron-send-logging-checks.py

```diff
@@ -0,0 +1,265 @@
+        self.es_curl = "curl --cert {}cert --key {}key --cacert {}ca -XGET ".format(es_cert, es_cert, es_cert)
+
+    def parse_args(self):
+        ''' Parse arguments passed to the script '''
+        parser = argparse.ArgumentParser(description='OpenShift Cluster Logging Checker')
+        parser.add_argument('-v', '--verbose', action='store_true', default=None, help='Verbose output')
+        parser.add_argument('--debug', action='store_true', default=None, help='Debug?')
+
+        self.args = parser.parse_args()
+
+    def get_pods(self):
+        ''' Get all pods and filter them in one pass '''
+        pods = self.oc.get_pods()
+        for pod in pods['items']:
+            if 'component' in pod['metadata']['labels']:
+                # Get ES pods
+                if pod['metadata']['labels']['component'] == 'es':
+                    self.es_pods.append(pod)
+                elif pod['metadata']['labels']['component'] == 'fluentd':
+                    self.fluentd_pods.append(pod)
+
+    # Disabling all of these so we do not have to loop over pods again
+    # These will get their own proper setup once we have the sidecar pod
+    #pylint: disable=too-many-locals
+    #pylint: disable=too-many-branches
+    #pylint: disable=too-many-statements
+    def check_elasticsearch(self):
+        ''' Various checks for elasticsearch '''
+        es_status = {}
+        es_status['single_master'] = None
+        es_master_name = None
+        es_status['pods'] = {}
+        for pod in self.es_pods:
+            pod_dc = pod['metadata']['labels']['deploymentconfig']
+            pod_name = pod['metadata']['name']
+            es_status['pods'][pod_dc] = {}
+
+
+            # exec into a pod and get cluster health
+            try:
+                curl_cmd = "{} 'https://localhost:9200/_cluster/health?pretty=true'".format(self.es_curl)
+                cluster_health = "exec -ti {} -- {}".format(pod_name, curl_cmd)
+                health_res = self.oc.run_user_cmd(cluster_health)
+
+                if health_res['status'] == 'green':
+                    es_status['pods'][pod_dc]['elasticsearch_health'] = 2
+                elif health_res['status'] == 'yellow':
+                    es_status['pods'][pod_dc]['elasticsearch_health'] = 1
+                else:
+                    es_status['pods'][pod_dc]['elasticsearch_health'] = 0
+            except:
+                # The check failed so ES is in a bad state
+                es_status['pods'][pod_dc]['elasticsearch_health'] = 0
+
+            # Exec into the pod and get diskspace, this will be cleaner once we
+            # have time to build a sidecar pod out of this.
+            try:
+                disk_used = 0
+                disk_free = 0
+                trash_var = 0
+
+                disk_output = self.oc.run_user_cmd("exec -ti logging-es-dy48r5sl-3-1po1n -- df").split(' ')
+                disk_output = filter(None, disk_output)
```

Comment:
This is a pylint check. Here's more detail:
http://stackoverflow.com/questions/21806765/how-is-using-filter-a-bad-builtin